.github/ISSUE_TEMPLATE/2_bug_report_macos.yml

```diff
@@ -0,0 +1,71 @@
+name: "Report a macOS Issue"
+description: "Create a report for a runtime related macOS Issue"
+labels: ["os:mac"]
+body:
+- type: textarea
+  attributes:
+      label: "Important Information"
+      description: >
+          - mpv bersion
+           - macOS bersion
+           - Source of the mpv binary or bundle
+           - If known which version of mpv introduced the problem
+           - Possible screenshot or video of visual glitches
+
+          If you're not using git master or the latest release, update.
+          Releases are listed [here](https://github.com/mpv-player/mpv/releases).
+      value: |-
+          - mpv version:
+          - macOS version:
+          - Source:
+          - Introduced in version:
+  validations:
+      required: true
+- type: textarea
+  attributes:
+      label: "Reproduction Steps"
+      description: >
+          Try to reproduce your issue with `--no-config` first. If it isn't reproducible
+          with `--no-config` try to find out which option or script causes your issue.
+
+          If the issue is performance-related, try to reproduce it with `--no-config
+          --profile=fast`, which is designed to work on lower-end mobile devices.
+          Additionally, enable hardware decoding with `--hwdec=yes` or the `Ctrl+h` shortcut.
+
+          Describe the reproduction steps as precise as possible. It's very likely that
+          the bug you experience wasn't reproduced by the developer because the workflow
+          differs from your own.
+  validations:
+      required: true
+- type: textarea
+  attributes:
+      label: "Expected Behavior"
+  validations:
+      required: true
+- type: textarea
+  attributes:
+      label: "Actual Behavior"
+  validations:
+      required: true
+- type: textarea
+  attributes:
+      label: "Log File"
+      description: >
+          Make a log file made with `-v -v` or `--log-file=output.txt`, attach it to
+          the issue. If you use the Bundle, a default log is created for your last run at
+          `~/Library/Logs/mpv.log`. You can jump to that file via the `Help > Show log File…`
+          menu.
+
+          In the case of a crash, please provide the macOS Crash Report (Backtrace).
+
+          Without the log file, this issue will be closed for ignoring the issue template.
+  validations:
+      required: true
+- type: textarea
+  attributes:
+      label: "Sample Files"
+      description: >
+          Sample files needed to reproduce this issue can be attached to the issue
+          (preferred), or be uploaded to https://0x0.st/ or similar sites.
```

Comment:
https://0x0.st/ is good, but the retention time of files make it not really the best choice for reporting issues which can easily outlive hosted files.

Reply:
i can't really change this though. without some better suggestion i am not sure what to do?

Reply:
ok